services/web/server/tests/unit/with_dbs/02/test_projects_crud_handlers__list_with_query_params.py

```diff
@@ -251,3 +252,111 @@ async def test_list_projects_with_search_parameter(
     assert data["_meta"]["limit"] == 1
     assert data["_links"]["next"].endswith("/v0/projects?search=oda&offset=1&limit=1")
     assert data["_links"]["last"].endswith("/v0/projects?search=oda&offset=1&limit=1")
+
+
+_alphabetically_ordered_list = ["a", "b", "c", "d", "e"]
+
+
+@pytest.mark.parametrize(*standard_user_role())
+async def test_list_projects_with_order_by_parameter(
+    client: TestClient,
+    logged_user: UserDict,
+    expected: ExpectedResponse,
+    fake_project: ProjectDict,
+    tests_data_dir: Path,
+    osparc_product_name: str,
+    project_db_cleaner,
+    mock_catalog_api_get_services_for_user_in_product,
+):
+    projects_info = [
+        _ProjectInfo(
+            uuid="aaa0eca3-d210-4db6-84f9-63670b07176b",
+            name="d",
+            description="c",
+        ),
+        _ProjectInfo(
+            uuid="cccef868-fe1b-11ed-b038-cdb13a78a6f3",
+            name="b",
+            description="e",
+        ),
+        _ProjectInfo(
+            uuid="eee66c12-fe1b-11ed-b038-cdb13a78a6f3",
+            name="a",
+            description="a",
+        ),
+        _ProjectInfo(
+            uuid="ddd32426-fe1b-11ed-b038-cdb13a78a6f3",
+            name="c",
+            description="b",
+        ),
+        _ProjectInfo(
+            uuid="bbb7aff6-fe1b-11ed-b038-cdb13a78a6f3",
+            name="e",
+            description="d",
+        ),
+    ]
+
+    user_projects = []
+    for project_ in projects_info:
+        project_data = deepcopy(fake_project)
+        project_data["name"] = project_.name
+        project_data["uuid"] = project_.uuid
+        project_data["description"] = project_.description
+
+        user_projects.append(
+            await _new_project(
+                client,
+                logged_user["id"],
+                osparc_product_name,
+                tests_data_dir,
+                project_data,
+            )
+        )
+
+    # Order by uuid ascending
+    base_url = client.app.router["list_projects"].url_for()
+    url = base_url.with_query(
+        order_by=json.dumps({"field": "uuid", "direction": "asc"})
```

Comment:
Q: why did you allow sorting by uuid? Is there any special use case you h ave in mind? IMO we should not expose features we are not going to use

Reply:
I actually like that one could filter by uuid, especially for debugging purpose. People are used to give us the project uuid

Reply:
Its not a special feature. We expose uuid in API, therefore I think is valid to give the possiblity to sort on any field that we expose.